vms/vm-under-test/scripts/first-boot

```diff
@@ -21,3 +21,9 @@ set -e
 
 mkdir /mnt/huge
 mount /mnt/huge --source nodev -t hugetlbfs -o pagesize=1GB
+
+echo isolated_cores=1-2 >> /etc/tuned/realtime-virtual-guest-variables.conf
+echo isolate_managed_irq=Y >> /etc/tuned/realtime-virtual-guest-variables.conf
+tuned-adm profile realtime-virtual-guest
+
+reboot
```

Comment:
Is there a way to avoid this reboot here?
I'm afraid that it will mess with the checkup's detection of when the VM had completed booting.